cloudbuild.yaml

```diff
@@ -49,13 +49,9 @@ steps:
 - name: 'gcr.io/$PROJECT_ID/e2e-go-mod-ginkgo:0.4.28'
   entrypoint: 'bash'
   env:
-  - 'ACK_GINKGO_RC=true'
```

Comment:
These are defined in the Makefile. By specifying necessary variables there, when devs run the make targets locally, they will benefit.